src/library/Uno.Material/Controls/Ripple.cs

```diff
@@ -0,0 +1,129 @@
+﻿/*
+ * Starting implementation acknowledgement from project https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/2740f14a814896d42032ae0013b765a8a0ec04c3/MaterialDesignThemes.Uwp/Ripple.cs
```

Comment:
Could move this part right before the `class`.

Reply:
done